frontend/src/audio.js

```diff
@@ -18,9 +18,9 @@ document.addEventListener('alpine:initialized', () => {
         contentDetails: {
           title: article.dataset.audioTitle,
           dateForHumans: article.dataset.dateForHumans,
-          dateUncertain: article.dataset.dateUncertain,
+          dateUncertain: JSON.parse(article.dataset.dateUncertain),
           location: article.dataset.location,
-          locationUncertain: article.dataset.locationUncertain,
+          locationUncertain: JSON.parse(article.dataset.locationUncertain),
```

Comment:
In general, I don't like what we did. I mean the serialization and de-serialization of the object. That's why so many underwater stones.

Let's think the other way. What if we add parameters to the `audio-player` partial such as `fileId` and `contentDetails`? Then we can directly pass it as values from our `audio` template and pass nothing from the `audios` page (the home one). Let's try, it should work. We will not need the `dispatchEvent` then at all.